cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -12,10 +12,28 @@ class HttpCache
   # A map from script name to script level URL pattern.
   CACHED_SCRIPTS_MAP = {
       # We specify the "special case" routes here; standard routes are handled by the loop below.
-      'hourofcode' => '/hoc/*'
+      'hourofcode' => '/hoc/*',
+      'flappy' => '/flappy/*'
   }
-  %w(starwars starwarsblocks mc frozen gumball).each do |script_name|
-    CACHED_SCRIPTS_MAP[script_name] = "/s/#{script_name}/stage/1/puzzle/*"
+
+  # Cached scripts
+  %w(
+    starwars
+    starwarsblocks
+    mc
+    frozen
+    gumball
+    20-hour
+    course1
+    course2
+    course3
+    course4
+    playlab
+    artist
+    infinity
+    iceage
+  ).each do |script_name|
```

Comment:
no reason this list _should_ be alphabetized, as far as I can tell - we haven't organized script-names in alphabetical order anywhere in our codebase or in our levelbuilder interface, no need to start here.